app/javascript/src/utils/compiler/comments.js

```diff
@@ -1,3 +1,3 @@
 export function removeComments(joinedItems) {
-  return joinedItems.replaceAll(/\/\*[\s\S]*?\*\/|([^:]|^)\/\/.*$/gm, "")
+  return joinedItems.replaceAll(/\/\*[\s\S]*?\*\/|(?:(?<=^)|(?<=[^:]))\/\/.*$/gm, "")
```

Comment:
Nice! That second positive lookbehind could be a negative lookbehind for `:` https://regex101.com/r/iZ4DRL/1
```suggestion
  return joinedItems.replaceAll(/\/\*[\s\S]*?\*\/|(?:(?<=^)|(?<!:))\/\/.*$/gm, "")
```

Or even better IMO, join both positive lookbehinds and remove the wrapping non-capturing group: https://regex101.com/r/rBo8D2/1
```suggestion
  return joinedItems.replaceAll(/\/\*[\s\S]*?\*\/|(?<=^|[^:])\/\/.*$/gm, "")
```

Reply:
Looks good to me 👍 i trust your regexing more than my own (well, chatGPT's) 😅